sdks/python/apache_beam/examples/inference/README.md

```diff
@@ -24,7 +24,7 @@ API. <!---TODO: Add link to full documentation on Beam website when it's publish
 
 ## Prerequisites
 
-You must have `apache-beam>=2.40.0` installed in order to run these pipelines,
+You must have `apache-beam>=2.40.0` or greater installed in order to run these pipelines,
 because the `apache_beam.examples.inference` module was added in that release.
 ```
 pip install apache-beam==2.40.0
```

Comment:
Should we have `apache-beam>=2.40.0` to be consistent with the above?

Reply:
Good call, I updated to do this